.github/workflows/validate.yaml

```diff
@@ -26,9 +26,17 @@ jobs:
         with:
           go-version-file: ./src/github.com/${{ github.repository }}/go.mod
 
+      - name: Setup yq
+        env:
+          YQ_VERSION: 3.4.0
+        run: |
+          sudo wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64 -O /usr/bin/yq
+          sudo chmod +x /usr/bin/yq
+          sudo mv /usr/bin/yq /usr/local/bin/yq
+
       - name: Regenerate all generated files
         working-directory: ./src/github.com/${{ github.repository }}
-        run: ./openshift/release/generate-release.sh ${{ github.base_ref }}
```

Comment:
It seems no according to the original PR since we don't use "$1" any more check here: https://github.com/openshift-knative/serving/pull/832/files#diff-9968a5e8c5ac80bfcb0ccaa76d57bb63568e94eddf378df926fccb6b43cb6be4.

Reply:
ok 👍